featurebyte/service/entity_relationship_extractor.py

```diff
@@ -150,7 +87,8 @@ def generate(self, entity_ids: List[ObjectId]) -> List[List[ObjectId]]:
         """
         reduced_entity_ids = self.reduce_entity_ids(entity_ids=entity_ids)
         entity_with_descendants_iterable = [
-            self.entity_id_to_descendant_ids[entity_id] | {entity_id}
+            self.entity_ancestor_descendant_mapper.entity_id_to_descendant_ids[entity_id]
+            | {entity_id}
             for entity_id in reduced_entity_ids
         ]
```

Comment:
Refactor this class by moving some functionalities into featurebyte/query_graph/model/entity_relationship_info.py.